### PR TITLE
feat: install zsh on unbuntu for developing

### PR DESCRIPTION
**Pull request by zhongwencool**

### CODE REVIEW

**Comment by id** [comment]
@zhongwencool missing `get-zsh.sh` script

**Comment by id** [comment]
But if you install zsh package, may be you don't need that script in the Dockerfile?

**Comment by zhongwencool** [comment]
> But if you install zsh package, may be you don't need that script in the Dockerfile?

The current bash is too basic, it's inconvenient to develop within it
I want to install ohmyzsh's plugins to make terminal experience delightful. :)
https://github.com/deluan/zsh-in-docker/blob/master/README.md

**Comment by id** [comment]
> > But if you install zsh package, may be you don't need that script in the Dockerfile?
> 
> The current bash is too basic, it's inconvenient to develop within it I want to install ohmyzsh's plugins to make terminal experience delightful. :) https://github.com/deluan/zsh-in-docker/blob/master/README.md

What I meant is since you install `zsh` package in Ubuntu already, why would you need `get-zsh.sh` script.
But now I see that `get-zsh.sh` actually installs oh-my-zsh, not zsh itself.

**Review by id — DISMISSED** [review]

File: Dockerfile [hunk 1]
```diff
@@ -9,14 +9,15 @@ ARG FDB_VERSION=7.3.27
...
+    /get-zsh.sh && \
     env FDB_VERSION=${FDB_VERSION} /get-fdb.sh && \
     env EMQTT_BENCH_REF=${EMQTT_BENCH_REF} /get-emqtt-bench.sh && \
     env LUX_REF=${LUX_REF} /get-lux.sh && \
-    rm /get-otp.sh /get-elixir.sh /get-fdb.sh /get-emqtt-bench.sh /get-lux.sh
+    rm /get-otp.sh /get-zsh.sh /get-elixir.sh /get-fdb.sh /get-emqtt-bench.sh /get-lux.sh
 
 WORKDIR /
-CMD [ "/bin/bash" ]
+CMD [ "/bin/zsh" ]
```
Comment (id) [inline Dockerfile:23]: This will break non-Ubuntu images.

File: ubuntu16.04/Dockerfile [hunk 2]
```diff
@@ -89,5 +90,6 @@ RUN apt-get clean \
 # Elixir complains if runs without UTF-8
 ENV LC_ALL=C.UTF-8
 ENV LANG=C.UTF-8
+ENV ERL_AFLAGS="-kernel shell_history enabled"
```
[the remark this answers is not in the document]
Comment (id) [reply ubuntu16.04/Dockerfile:93]: Or even better, move it to the main `./Dockerfile`

**Comment by id** [comment]
To summarize my comments: I don't think you need to change any distro-specific Dockerfiles.
`get-zsh.sh` will install `zsh` in the final image for all distros + oh-my-zsh, and you can just add `ENV ERL_AFLAGS="-kernel shell_history enabled"` there.

**Review by id — APPROVED** [review]

**Review by id — DISMISSED** [review]

**Comment by id** [comment]
https://github.com/deluan/zsh-in-docker/pull/26

**Comment by zhongwencool** [comment]
I've noticed that this repo is rarely updated. Why don't we just copy that script directly into get-zsh.sh and maintain it ourselves?
It's MIT license.

**Comment by id** [comment]
I'm testing this now https://github.com/emqx/emqx-builder/pull/112.
If it works, we can just switch to our fork.

**Review by thalesmg — APPROVED** [review]

**Pull request merged**